c7n/resources/rdscluster.py

```diff
@@ -399,6 +401,59 @@ def process(self, clusters):
                 client.exceptions.InvalidDBClusterStateFault)
 
 
+@RDSCluster.action_registry.register('modify-db-cluster')
+class ModifyDbCluster(BaseAction):
+    """Modifies an RDS instance based on specified parameter
+    using ModifyDbInstance.
+
+    'Immediate" determines whether the modification is applied immediately
+    or not. If 'immediate' is not specified, default is false.
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: disable-db-cluster-deletion-protection
+                resource: rds-cluster
+                filters:
+                  - DeletionProtection: true
+                  - PubliclyAccessible: true
+                actions:
+                  - type: modify-db-cluster
+                    attributes:
+                        CopyTagsToSnapshot: true
+                        DeletionProtection: false
+    """
+
+    schema = type_schema(
+        'modify-db-cluster',
+        immediate={"type": 'boolean'},
```

Comment:
immediate isn't being used by the implementation

Reply:
Got rid of immediate from the schema, such that attributes should be able to handle it